nutanix-cloud-native/cluster-api-provider-nutanix
# Feat/projects

**What this PR does / why we need it**:
Allows CAPI VMs to be added to a project inside of PC. The project needs to be created manually.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
n/a

**How Has This Been Tested?**:
- Create a project in PC
- Create a cluster.yml file and add the project reference (using name or uuid)
- Create CAPI cluster
- Go to the project in PC and verify if the VMs are added to the project

**Special notes for your reviewer**:
None

**Release note**:

```release-note
- Allows adding CAPI cluster VMs to Projects
   Note: Projects need to be created manually 
```
